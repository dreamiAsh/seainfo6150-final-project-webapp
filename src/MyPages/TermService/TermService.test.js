import React from "react";
import { render, fireEvent } from "@testing-library/react";
import TermService from "./TermService.jsx";

it("renders the TermService component correctly", () => {
  const { container } = render(<TermService />);
  expect(container).toMatchSnapshot();
});