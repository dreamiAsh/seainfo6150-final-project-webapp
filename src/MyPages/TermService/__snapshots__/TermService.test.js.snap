// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the TermService component correctly 1`] = `
<div>
  <div
    class="box"
  >
    <div
      class="box"
    >
      <div
        class="imgBox"
      >
        <img
          alt="poster image"
          class="img"
          src="https://covid19.lacounty.gov/wp-content/uploads/GettyImages-1128687123-1024x683.jpg"
        />
      </div>
      <h5
        class="title"
      >
        Terms of Service Agreement
      </h5>
      <p
        class="content"
      >
        PLEASE READ THIS TERMS AND CONDITIONS AGREEMENT ("AGREEMENT") CAREFULLY. THIS AGREEMENT IS A LEGAL CONTRACT BETWEEN YOU ("USER") AND MEREDITH CORPORATION ("MEREDITH," "COMPANY," "WE," OR "US"). SECTION 15 OF THIS AGREEMENT IS AN ARBITRATION CLAUSE THAT REQUIRES MOST DISPUTES BETWEEN US TO BE RESOLVED THROUGH BINDING AND FINAL ARBITRATION INSTEAD OF IN COURT. SEE SECTION 15 FOR MORE INFORMATION REGARDING THIS ARBITRATION CLAUSE, AND HOW TO OPT OUT.
By accessing or using any website with an authorized link to this Agreement ("Website"), registering an account or accessing or using any content, information, services, features or resources available or enabled via the Website (collectively with the Website, the "Services"), clicking on a button or taking another action to signify your acceptance of this Agreement, you: (1) agree to be bound by this Agreement and any future amendments and additions to this Agreement as published through the Services; (2) represent you are of legal age in your jurisdiction of residence to form a binding contract; and (3) represent that you have the authority to enter into this Agreement personally and, if applicable, on behalf of any company, organization or other legal entity on whose behalf you use the Services. Except as otherwise provided herein, if you do not agree to be bound by this Agreement, you may not access or use the Services. Your use of the Services is also subject to any additional terms, conditions and policies that we separately post on the Services ("Supplemental Terms") which are incorporated by reference into this Agreement. Some of the specialized features and tools accessible through our Services are provided by third-party companies pursuant to their own separate Terms of Service (“Third-Party Terms”) that differ from ours.  By using such third-party features and tools, you agree that your relationships with the third-party service providers will be governed by the applicable Third-Party Terms. Subject to Section 15.9 of this Agreement, Meredith reserves the right to modify this Agreement or its policies relating to the Services at any time, effective upon posting of an updated version of this Agreement or any applicable Supplemental Terms on the applicable Services. You should regularly review this Agreement, as your continued use of the Services after any such changes constitutes your agreement to such changes. 1. Registration. When registering an account for the Services ("Account"), you agree to provide only true, accurate, current and complete information requested by the registration form (the "Registration Data") and to promptly update the Registration Data thereafter as necessary. You represent that you are not barred from using the Services under any applicable law and that you will be responsible for all activities that occur under your Account. You agree to monitor your Account to restrict its use by minors and other unauthorized users and agree not to share your Account or password with anyone. You further agree to notify Meredith immediately of any unauthorized use of your password or any other breach of the security of your Account and to exit from your Account at the end of each session. You agree not to create an Account using a false identity or alias or if you previously have been banned from using any of the Services. You further agree that you will not maintain more than one Account for the same Meredith service at any given time. Meredith reserves the right to remove or reclaim any usernames at any time and for any reason. You acknowledge and agree that you have no ownership or other property interest in your Account and that all rights in and to your Account are owned by and inure to the benefit of Meredith. 2. Order Process. 2.1 Order Acceptance. Each part of any order that you submit to Meredith constitutes an offer to purchase. If you do not receive a message from Meredith confirming receipt of your order, please contact our Customer Service department before re-entering your order. 
      </p>
    </div>
  </div>
</div>
`;
